go.mod

```diff
@@ -10,3 +10,5 @@ require (
 	modernc.org/strutil v1.0.0 // indirect
 	modernc.org/xc v1.0.0
 )
+
+go 1.13
```

Comment:
I'm not sure what this implies and whether this should go in. The go tool did this by itself.

Reply:
This is fine.

Reply:
Ok, I'll keep it.